docs/src/content/docs/sdk/wallet-get-info.mdx

```diff
@@ -93,8 +93,37 @@ For JavaScript, run `node path/to/directory/index.js`. <LinkOut href='https://gi
 
 <p><LinkOut href='https://github.com/interledger/open-payments-php-snippets/blob/main/src/Command/WalletAddress/PublicGetWalletAddress.php'>View full source</LinkOut></p>
 </TabItem>
-</Tabs>
 
+<TabItem label="Go" icon='seti:go'>
+<LinkButton href="https://github.com/interledger/open-payments-go/blob/main/README.md" target="_blank" icon="external" iconPlacement="start">Prerequisites</LinkButton>
+
+<ChunkedSnippet
+  source="wallet-address/wallet-address-get.go"
+  l="go"
+  chunk="1"
+/>
+
+<ChunkedSnippet
+  source="wallet-address/wallet-address-get.go"
+  l="go"
+  chunk="2"
+/>
+
+<ChunkedSnippet
+  source="wallet-address/wallet-address-get.go"
+  l="go"
+  chunk="3"
+/>
+
+<ChunkedSnippet
+  source="wallet-address/wallet-address-get.go"
+  l="go"
+  chunk="4"
+/>
+
+<p><LinkOut href='https://github.com/interledger/open-payments/blob/main/snippets/go/wallet-address/wallet-address-get.go'>View full Go source</LinkOut></p>
```

Comment:
optimistic. these links dont exist yet - they will when this gets merged in.

alternatively, we could exclude these links and add them after its merged in. Seems like not a big deal to leave them and then just check/fix them after merging in.